client/public/index.html

```diff
@@ -20,6 +20,7 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
     <title>MDN Web Docs</title>
+    <meta name="description" content="The MDN Web Docs site provides information about Open Web technologies including HTML, CSS, and APIs for both Web sites and progressive web apps. It also has some developer-oriented documentation for Mozilla products, such as Firefox Developer Tools.">
```

Comment:
I would suggest removing the second sentence: I don't think internal Mozilla docs or Firefox devtools are going to make it into stumptown.

Reply:
Done.